docs/static/settings/configuration-management-settings.asciidoc

```diff
@@ -56,6 +56,11 @@ authenticate for accessing the configuration data.
 The username you specify here should have the `logstash_admin` role, which
 provides access to `.logstash-*` indices for managing configurations.
 
+`xpack.management.elasticsearch.proxy`::
```

Comment:
Remove `xpack.` prefix

Reply:
All the other settings are still prefixed with `xpack.` in the 7.x branch. Do you want I remove all of them ?

Reply:
Sorry, no for the `management ` only for `monitoring` . so this is good